tutorials/clinical/10_ieeg_localize.py

```diff
@@ -320,17 +321,22 @@ def plot_overlay(image, compare, title, thresh=None):
 
 # sphinx_gallery_thumbnail_number = 8
 
-# get native to head trans
-fsaverage_trans = mne.channels.compute_native_head_t(montage_warped)
+# first we need to add fiducials so that we can define the "head" coordinate
+# frame in terms of them (with the origin at the center between LPA and RPA)
+montage_warped.add_estimated_fiducials('fsaverage', subjects_dir)
+
+# compute the head<->mri ``trans`` now using the fiducials
+template_trans = mne.channels.compute_native_head_t(montage_warped)
 
-# set new montage
+# now we can set the montage and, because there are fiducials in the montage,
+# the montage will be properly transformed to "head" coordinates when we do
+# (this step uses ``template_trans`` but it is recomputed behind the scenes)
 raw.set_montage(montage_warped)
 
 # plot the resulting alignment
-renderer = mne.viz.backends.renderer.create_3d_figure(**fig_kwargs)
-fig = mne.viz.plot_alignment(raw.info, fsaverage_trans, 'fsaverage',
-                             fig=renderer.figure, **al_kwargs)
-mne.viz.set_3d_view(fig, **view_kwargs)
+brain = mne.viz.Brain('fsaverage', **brain_kwargs)
+brain.add_sensors(raw, trans=template_trans)
```

Comment:
```suggestion
brain.add_sensors(raw.info, trans=template_trans)
```
seems more natural

Reply:
Sure, it will support data viewing in the future as I understand but good for now